logging/google/cloud/logging/_gax.py

```diff
@@ -270,11 +272,13 @@ def sink_update(self, project, sink_name, filter_, destination):
         path = 'projects/%s/sinks/%s' % (project, sink_name)
         sink_pb = LogSink(name=path, filter=filter_, destination=destination)
         try:
-            self._gax_api.update_sink(path, sink_pb, options=options)
+            sink_pb = self._gax_api.update_sink(path, sink_pb, options=options)
```

Comment:
@lukesneeringer @tseaver This is a change I'm not sure I should make here. But I noticed that we are throwing away the "updated" protobuf and there may be small changes (e.g. timestamps) between the local one and the response.